target/linux/sunxi/image/cortex-a7.mk

```diff
@@ -31,6 +31,16 @@ define Device/friendlyarm_nanopi-m1-plus
 endef
 TARGET_DEVICES += friendlyarm_nanopi-m1-plus
 
+define Device/friendlyarm_nanopi-r1
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi R1
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-usb2 kmod-usb-net kmod-usb-net-rtl8152 \
+  kmod-brcmfmac kmod-leds-gpio kmod-ledtrig-heartbeat wpad-basic \
```

Comment:
Use single-tab indent for the second and third line of DEVICE_PACKAGES.

Reply:
issue was with VS Code editor not adding tab space. Single-tab indent added.